test/parallel/test-tls-env-extra-ca-file-load.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+// Flags: --expose-internals
+
+const common = require('../common');
+
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+const assert = require('assert');
+const tls = require('tls');
+const fixtures = require('../common/fixtures');
+const { internalBinding } = require('internal/test/binding');
+const binding = internalBinding('crypto');
+
+const { fork } = require('child_process');
+
+// This test ensures that extra certificates are loaded at startup.
```

Comment:
Could you implement a separation between main and child code. This is often used:
```js
if (process.argv[2] !== 'child') {
} else {
...
fork(__filename, ['child'], { env: processEnv })
...
}
```

Reply:
Done.